content/en/docs/languages/python/libraries.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Using instrumentation libraries
+linkTitle: Libraries
+weight: 40
+cSpell:ignore: HTTPX httpx instrumentor uninstrument
+---
+
+{{% docs/languages/libraries-intro "Python" %}}
+
+## Use instrumentation libraries
+
+If a library does not ship with native OpenTelemetry support, you can use
+[instrumentation libraries](/docs/specs/otel/glossary/#instrumentation-library)
+to generate telemetry data for a library or framework.
+
+For example,
+[the instrumentation library for HTTPX](https://pypi.org/project/opentelemetry-instrumentation-httpx/)
+automatically creates [spans](/docs/concepts/signals/traces/#spans) based on
+HTTP requests.
+
+## Setup
+
+You can install each instrumentation library separately using pip. For example:
+
+```sh
+pip install opentelemetry-instrumentation-{integration}
+```
+
+In the previous example, `{integration}` is the name of the instrumentation.
```

Comment:
I believe we use the term [instrumented library](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/glossary.md#instrumented-library) instead of `integration`.

Reply:
Fixed!